packages/flutter/test/rendering/view_chrome_style_test.dart

```diff
@@ -0,0 +1,255 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
```

Comment:
info: I couldn't use `view_test.dart` file because `TestRenderingFlutterBinding` it is used there. Those tests uses `TestWidgetsFlutterBinding`. 
Different types of bindings can't be used in the same file.